src/models/campaign.ts

```diff
@@ -523,7 +524,7 @@ export default class Campaign {
             },
         });
 
-        const namesFilters = dashboards.map(d => `name:like:${d.name.replace("_DASHBOARD", "")}`);
+        const namesFilters = dataSets.map(d => `name:like$:${d.name}`);
```

Comment:
Maybe we should filter further the category options (JS filter) by an extra "Team N - NAME" check. If that's the case, with a function in `Teams.ts` so logic is kept there.

Reply:
Something else I've noticed: 

https://github.com/EyeSeeTea/vaccination-app/pull/198/files/3ac7eccdb01ad80143c04d38a1a3c552c18aa118#diff-7aeb121974ad1acf2c873b9d41b02ae2L529

If `namesFilters` is empty, this call with empty filters will fail (409)